docs/group_quick_starts/gc_account_deletion.rst

```diff
@@ -0,0 +1,21 @@
+Account Deletion
+================================
+
+You can now delete your account without asking help from Solace Cloud support. However, this feature is limited to standard accounts only.
+
+In the below image, under Account tab, the Advance Setting link is been added to provide option for deleting account.
+
+.. figure:: ../img/delete_account_advanced_settings.png
+
+
+.. figure:: ../img/delete_account_popup.png
+
+After clicking the Delete Account button you would get a confirmation pop up which will inform you about all the impacts post deletion of your account.
+
+.. figure:: ../img/delete_account_confirmation_popup.png
+
+You would require to enter your password to ensure that it is an intentional deletion and voila, the account is deleted.
```

Comment:
I don't think you need this paragraph or the image.

Reply:
As discussed took out the other screenshot which shows only Delete button with password.